### DISCUSSION TITLE
valve Lidl 368308_2010

### DISCUSSION THREAD

**Discussion by picasoaaa** [Koenkk/zigbee2mqtt#9947] [Devices]
After updating 1.22.1-1
valve sends this, restart after error

Zigbee2MQTT:info  2021-12-02 08:20:04: MQTT publish: topic 'zigbee2mqtt/hlavice leva', payload '{"auto_lock":"AUTO","away_mode":null,"away_preset_days":0,"away_preset_temperature":{"0":6,"1":34,"10":96,"11":42,"12":96,"13":34,"14":96,"15":42,"16":96,"17":34,"2":24,"3":42,"4":36,"5":34,"6":68,"7":42,"8":92,"9":34},"battery_low":{"0":2,"1":34,"10":96,"11":42,"12":96,"13":34,"14":96,"15":42,"16":96,"17":34,"2":24,"3":42,"4":36,"5":34,"6":68,"7":42,"8":92,"9":34},"binary_one":null,"binary_two":null,"boost_time":42,"child_lock":null,"comfort_temperature":false,"current_heating_setpoint":0,"current_heating_setpoint_auto":"1.0","detectwindow_temperature":null,"detectwindow_timeminute":null,"eco_temperature":null,"holidays":[{"hour":5,"minute":34,"temperature":24},{"hour":42,"minute":36,"temperature":34},{"hour":4,"minute":42,"temperature":92},{"hour":34,"minute":96,"temperature":42},{"hour":32,"minute":34,"temperature":96},{"hour":42,"minute":96,"temperature":34}],"linkquality":96,"local_temperature_calibration":null,"max_temperature":{"0":20,"1":2,"2":29,"3":23,"4":0,"5":34,"6":0,"7":1},"min_temperature":34,"position":{"0":1,"1":116,"10":90,"11":90,"12":90,"13":90,"14":90,"15":90,"16":90,"17":90,"2":8,"3":8,"4":8,"5":8,"6":8,"7":8,"8":8,"9":90},"voltage":null,"window_detection":"OFF","window_detection_params":{},"workdays":[{"hour":4,"minute":34,"temperature":24},{"hour":42,"minute":36,"temperature":34},{"hour":4,"minute":42,"temperature":92},{"hour":34,"minute":96,"temperature":42},{"hour":32,"minute":34,"temperature":96},{"hour":42,"minute":96,"temperature":34}]}'
Error: ENOENT: no such file or directory, open 'data/tuya.dump.txt'
[08:20:04] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.22.1 start

**Comment by gmarchart** [comment]
yes, have the same issue - always when set something in climawidget m2z restarts, also local temp, calibrate temp did not work proberly for me.

that is allready discussed -> https://github.com/Koenkk/zigbee2mqtt/issues/9509#issue-1045514277

**Reply by picasoaaa** [reply]
Does anyone have a valve in the last be problem?

**Reply by gmarchart** [reply]
At the moment I use latest m2z with an external converter. Without external converter I got m2z restarts every time I chance something on valve.

With m2z edge which should work I have the problem that every 24?! Hours the time and date on the valve is wrong. That means no automatic schedule works after one day latest.  I guess set or reseted by m2z. This not happend with the external converter and official m2z.

I am a little bit frustrating with the chances on m2z for this valve. I guess they only assume that tuya valve and this silvercrest valve should have the same dp's. But it isn't....

**Reply by picasoaaa** [reply]
It behaves exactly as you described it, when I move the head, it causes an mqtt error and starts again.

**Reply by gmarchart** [reply]
i use that external converter - it works good -> https://github.com/Lavisx/lidl-valve-z2m/blob/main/lidl_radiator_valve.js


this external converter with the official m2z latest version. with this setup also the time/date on the valve is not touched and always correct - with the m2z-edge version i got always a wrong time/date after 24h ?! on the valve and then the automatic schedule does the changes of temp on a wrong time ...  for me important because i use the internal valve schedule to change the temperatur on a specific schedule...

**Reply by gmarchart** [reply]
see #10540 - works now in edge
